We do not need to check coverge for this project.

--- typeguard-4.4.4/pyproject.toml.orig
+++ typeguard-4.4.4/pyproject.toml
@@ -41,7 +41,6 @@
 
 [dependency-groups]
 test = [
-    "coverage[toml] >= 7",
     "pytest >= 7",
     'mypy >= 1.2.0; python_implementation != "PyPy"',
 ]
@@ -104,7 +103,7 @@
 requires = ["tox >= 4.22"]
 
 [tool.tox.env_run_base]
-commands = [["python", "-m", "coverage", "run", "-m", "pytest", { replace = "posargs", extend = true }]]
+commands = [["python", "-m", "pytest", { replace = "posargs", extend = true }]]
 package = "editable"
 dependency_groups = ["test"]
 
